### DISCUSSION TITLE
being able to resize images in match

### DISCUSSION THREAD

**Discussion by dark293** [octgn/OCTGN#2224] [Ideas]
Hi there, so I am on an ultra-wide 5120x1440 monitor, and it makes playing some card games rough because the card images on mouse over are based on window size, so if I play at fullscreen, they are tiny. My window for decent readability is small. So just asking if we could have an option to resize the Images in game.

**Comment by brine** [comment]
There's an "undock card preview" in-game menu option which pulls the image preview into its own resizeable window.  This may be the best solution for an unusual window size.

OCTGN will shrink-to-fit the card preview so that it fits within the height of the table.  However, it won't stretch/enlarge the image if the source image is smaller than the available space.

Could you provide a screenshot of what the OCTGN window looks like with your preferred setup?  There may be some ways to optimize it for ultra-wide screens.

**Reply by dark293** [reply]
The card preview helps 
![Screenshot 2023-08-16 014619](https://github.com/octgn/OCTGN/assets/18541042/3a1804c5-cea0-47f0-b288-6a9201bf5ebb)
I can't exactly get an in game pic since if I move mouse off it will stop but yeah after a certain size the card images shrink
